src/ui-config/marketsConfig.tsx

```diff
@@ -699,4 +701,20 @@ export const marketsData: {
       COLLECTOR: AaveV3Linea.COLLECTOR,
     },
   },
+  [CustomMarket.proto_sonic_v3]: {
+    marketTitle: 'Sonic',
+    market: CustomMarket.proto_sonic_v3,
+    chainId: ChainId.sonic,
+    v3: true,
+    // subgraphUrl: ``, // TODO: add subgraph url (waiting for fix)
```

Comment:
@grothem we ready to roll for this?

Reply:
can't publish subgraphs yet, Safe api still down